app/views/layouts/listnav/_host.html.haml

```diff
@@ -119,6 +119,15 @@
             :record_id     => @record.id,
             :title         => _("Show %{host} drift history") % {:host => host_title})
 
+        - if role_allows?(:feature => "physical_server_show") && !@record.physical_server.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "physical_servers")}: #{@record.physical_server.name}",
```

Comment:
I  believe it's the `:table => physical _servers`, I tested `:table => physical _server` and it looked good.